common/script/aria.js

```diff
@@ -425,15 +432,19 @@ require(["core/pubsubhub"], function( respecEvents ) {
                             for (var k = 0; k < sortedList.length; k++) {
                                 var role = sortedList[k];
                                 var req = "";
+                                var dep = "";
                                 if (role.required) {
                                     req = " <strong>(required)</strong>";
                                 }
+                                if (role.deprecated) {
+                                    dep = " <strong>(deprecated in ARIA 1.2)</strong>"
```

Comment:
I can't help thinking that authors could easily misunderstand this, and think that the attribute itself has been deprecated (not just its use as a global). Suggest adding "as global" to hopefully help mitigate any confusion.

```suggestion
                                    dep = " <strong>(deprecated as global in ARIA 1.2)</strong>"
```